lib/pages/league/widgets/dialogs/rules_dialog.dart

```diff
@@ -0,0 +1,365 @@
+import 'package:ax_dapp/service/custom_styles.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:toggle_switch/toggle_switch.dart';
```

Comment:
I do not think we need this package since Flutter has their own version. It is called [ToggleButton](https://api.flutter.dev/flutter/material/ToggleButtons-class.html)